README.md

```diff
@@ -35,7 +35,8 @@ Create a local.settings.json file with the following content:
       "AzureWebJobsStorage": "UseDevelopmentStorage=true",
       "QueueConnectionString": "UseDevelopmentStorage=true",
       "BlobConnectionString": "UseDevelopmentStorage=true",
-      "TableConnectionString": "UseDevelopmentStorage=true"
+      "TableConnectionString": "UseDevelopmentStorage=true",
+      "ServiceBusEventQueueName": "mit-queue-name"

```

Comment:
Rename to 'EventQueueName' since it's no longer ServiceBus (to avoid confusion), and the actual name we'll have to use will be 'rpa-mit-events' (to align with other repos) - although locally it doesn't make much difference

Reply:
agreed